datastore/src/test/java/io/spine/server/storage/datastore/DsFiltersShould.java

```diff
@@ -164,6 +168,28 @@ public void generate_filters_from_empty_params() {
         assertSize(0, filters);
     }
 
+    //TODO:2018-06-08:dmytro.kuzmin: re-write without mocks when null column filters are available.
```

Comment:
This has to be a reference to a `core-java` issue, which tells to implement `null` column filters.